src/lib/Behat/PageElement/ElementFactory.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement;
+
+use EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext;
+
+class ElementFactory
+{
+    /**
+     * Creates a Page object based on given Page Name.
```

Comment:
I know you know this, just so that it does not get lost: remember about amending PHPDocs 😛